docs/Versioning.md

```diff
@@ -24,18 +24,18 @@ Following table summarizes the relationship between the ONNX Runtime version and
 opset version implemented in that release. Please note the Backwards and Forward compatibility notes above.
 For more details on ONNX Release versions, see [this page](https://github.com/onnx/onnx/blob/master/docs/Versioning.md).
 
-| ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [WinML compatibility](https://docs.microsoft.com/en-us/windows/ai/windows-ml/)|
+| ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [WinML compatibility](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
```

Comment:
Would it be worthwhile to describe what "WinML compatibility" means? It may be a bit confusing what the relationship between ORT and WinML is wrt versioning.

Reply:
What about "WinML Availability"?